capstone/capapi/admin.py

```diff
@@ -18,17 +20,16 @@ def authenticate_user(modeladmin, request, queryset):
 
 class APIUserAdmin(admin.ModelAdmin):
     readonly_fields = ('date_joined', 'api_key')
-    list_display = ('email', 'last_name', 'first_name', 'api_key')
-    fields = (
+    list_display = (
         'email',
-        'first_name',
         'last_name',
-        'case_allowance',
-        'date_joined',
+        'first_name',
         'api_key',
-        'is_researcher',
+        'case_allowance_remaining',
+        'total_case_allowance',
     )
 
+    fields = deepcopy(list_display) + ('is_active', 'date_joined', 'activation_nonce', 'is_researcher')
```

Comment:
Seems like this works the same without deepcopy?

Reply:
yes why it absolutely does